### PR TITLE
Scroll through all cards on screen when arrows are clicked

### LINKED ISSUE #937: Scrolling through cards on CMS homepage should scroll all cards present on screen
**Actual Behaviour**

Currently, scrolling through cards on the CMS homepage isn't scrolling through an entire card. We're left somewhere in the middle of 2 cards which isn't desired.
Based on the reviews on https://github.com/fossasia/susi_skill_cms/pull/874, it was supposedly fixed, but apparently the issue still remains.

**Expected Behaviour**

Clicking on either of the 2 arrows should scroll through all the cards present on the screen, just like how it's done on Alexa.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Fixes #937 

Changes: Scroll through all cards present on screen when arrows are clicked.

Surge Deployment Link: https://pr-941-fossasia-susi-skill-cms.surge.sh
Alternate link: http://brash-decision.surge.sh/

Screenshots for the change:

Before:
scrollentirecardold

After:
scrollthroughallcardsonscreen

### CODE REVIEW

**Comment by Akshat-Jain** [comment]
@akshatnitd This seems to be a separate issue. This issue isn't caused by this PR. It's already there, you can check the same on https://skills.susi.ai. I checked the previous commits and it seems that this issue has been there since the horizontal scroll has been implemented for the metrics (https://github.com/fossasia/susi_skill_cms/pull/850). I think we can take this as a separate issue as this is not related to what this PR is about. Can you please review again? Thanks :-)

**Comment by akshatnitd** [comment]
Can you please check https://skills.susi.ai. When the screen is at full-width, all the skills show-up for me, on the horizontal scroll ?

**Comment by Akshat-Jain** [comment]
@akshatnitd Yes, but they show up in the deployment link I provided as well when the screen is at full width. I still think that all these issues should be taken as a separate issue, as these are unrelated to this PR. Thanks :-)

**Comment by Akshat-Jain** [comment]
@akshatnitd @DNS-404 Opened an issue for the same https://github.com/fossasia/susi_skill_cms/issues/946. Thanks :-)

File: src/components/SkillCardScrollList/SkillCardScrollList.js [hunk 1]
```diff
@@ -18,12 +18,36 @@ class SkillCardScrollList extends Component {
...
+
+  componentWillUnmount = () => {
+    window.removeEventListener('resize', this.updateWindowDimensions);
+  };
+
+  updateWindowDimensions = () => {
+    this.setState({
+      /* eslint-disable no-nested-ternary */
+      scrollCards:
+        window.innerWidth > 1400
```
Comment (akshatnitd) [inline src/components/SkillCardScrollList/SkillCardScrollList.js]: Remove `disable eslint` and go through this (https://stackoverflow.com/questions/12984191/alternative-for-multiple-if-statements) to avoid warnings/error.

File: src/components/SkillCardScrollList/SkillCardScrollList.js [hunk 2]
```diff
@@ -18,12 +18,36 @@ class SkillCardScrollList extends Component {
...
   }
 
-  componentDidMount() {
+  componentDidMount = () => {
     this.loadSkillCards();
-  }
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions);
+
+    $('.scrolling-wrapper').css({ width: '104%' });
```
Comment (akshatnitd) [inline src/components/SkillCardScrollList/SkillCardScrollList.js]: Why is this needed?

Comment (Akshat-Jain) [reply src/components/SkillCardScrollList/SkillCardScrollList.js]: This was done to make the arrow icon look better on full width, but I think I'll remove this from here and take care of this along with the other UI issues in https://github.com/fossasia/susi_skill_cms/issues/946. Thanks :-)

**Comment by Akshat-Jain** [comment]
@akshatnitd I've made the changes suggested by you. Unfortunately, I cannot get the Travis to pass. I've restarted it like 10 times already. It's failing every time. 

The tests are passing locally:
screenshot_2018-07-03 15 09 22_oelgfe

I've also updated the PR description with another surge link, which has the latest changes done for this PR. Please check. Thanks :-)

**Comment by mariobehling** [comment]
@Akshat-Jain We need travis to pass. Otherwise the project will not build in the gh-pages branch.
Please check how to resolve it.
